core/src/main/java/org/apache/brooklyn/enricher/stock/Transformer.java

```diff
@@ -44,9 +44,6 @@
     public static ConfigKey<Function<?, ?>> TRANSFORMATION_FROM_VALUE = ConfigKeys.newConfigKey(new TypeToken<Function<?, ?>>() {}, "enricher.transformation");
     public static ConfigKey<Function<?, ?>> TRANSFORMATION_FROM_EVENT = ConfigKeys.newConfigKey(new TypeToken<Function<?, ?>>() {}, "enricher.transformation.fromevent");
     
-    public Transformer() {
```

Comment:
I went back and forth for year on the implicit vs explicit argument. An argument would be that if it's there with no value, one may infer that there was an intention to do something but then the developer forgot. One convention made in other projects is to add a comment in the empty block that says something like "intentionally left blank". This applies even more to empty catch() blocks. My $0.02.